backend/src/services/secret-import/secret-import-service.ts

```diff
@@ -688,6 +725,51 @@ export const secretImportServiceFactory = ({
     }));
   };
 
+  const getImportsMultiEnv = async ({
+    path: secretPath,
+    environments,
+    projectId,
+    actor,
+    actorId,
+    actorAuthMethod,
+    actorOrgId,
+    search,
+    limit,
+    offset
+  }: Omit<TGetSecretImportsDTO, "environment"> & { environments: string[] }) => {
+    const { permission } = await permissionService.getProjectPermission({
+      actor,
+      actorId,
+      projectId,
+      actorAuthMethod,
+      actorOrgId,
+      actionProjectType: ActionProjectType.SecretManager
+    });
+    for (const environment of environments) {
+      ForbiddenError.from(permission).throwUnlessCan(
```

Comment:
Same here, I think @scott-ray-wilson did some internal flag check like in secret service to avoid throwing error